port: '/dev/ttyUSB0'
baudrate: 115200
timeout: 0.1

base_width: 0.26
ticks_per_meter: 4904.7
turn_direction: true

left_inverted: true
right_inverted: true

left_enc_inverted: true
right_enc_inverted: true

use_dynamic_acceleration: true
acceleration: 3000
deceleration: 5000
brake_accel:  7000
deceleration_time_limit: 500 #ms

update_period: 16.667  #ms

odom_topic: '/odom/wheel'



